flang/runtime/extensions.cpp

```diff
@@ -39,8 +40,10 @@ inline void CtimeBuffer(char *buffer, size_t bufsize, const time_t cur_time,
 #endif
 
 #if _REENTRANT || _POSIX_C_SOURCE >= 199506L
-// System is posix-compliant and has getlogin_r
+// System is posix-compliant and has getlogin_r and sleep
 #include <unistd.h>
+#elif _WIN32
+#include <windows.h> // for sleep
```

Comment:
Thank you for implementing this!

Have you considered using `std::this_thread::sleep_for` to avoid the need for Windows specific header file?

Reply:
Would that result in a call to the c++ lib?

Reply:
So far as I can tell, on my system (gnu libc++) `std::this_thread_sleep_for` is header-only. However, I haven't found any documentation indicating whether this is a safe assumption on other systems.

Reply:
I've updated to use std::this_thread::sleep_for for windows compatiability. Fingers crossed I didn't just add a dependency on libc++ on some obscure target.